www/content/workstation/upgrade_labs.md

```diff
@@ -0,0 +1,454 @@
++++
+title = "Upgrade Lab: Chef Infra Client 12 to 16"
+draft = false
+
+aliases = ["/upgrade_labs.html", "/upgrade_labs.html", "/upgrade_labs/", "/upgrade_labs/"]
+
+[menu]
+  [menu.workstation]
+    title = "Upgrade Lab: Chef Infra 12 to 16"
+    identifier = "chef_workstation/upgrade_labs.md Upgrade Chef"
+    parent = "chef_workstation"
+    weight = 35
++++
+
+Chef's Upgrade Lab provides an isolated cookbook development environment and in-line support to help you upgrade your system, so you can stop using legacy Chef Infra and start using modern Chef Infra.
+
+The Upgrade Lab provides a report of your existing nodes and cookbooks, so you'll know the scope of the work and you can identify a good place to start. Upgrade Lab works by capturing any node from the production environment and recreating it locally by generating a repository for that node, which provides you with a sandbox to work through upgrading and testing your cookbooks at a safe distance from your production environment.
+
+## About This Guide
+
+This guide covers the straightforward pattern of upgrading from Chef Infra Client 12 to Chef Infra Client 16---including upgrading Chef Infra Server as you go. While we think this is the smoothest path forward, it is not meant to exclude other approaches.
+
+Unless otherwise indicated, you'll run all commands in your local development environment.
+
+## Chef Upgrade Lab Requirements
+
+### System Architecture Requirements
+
+The Chef Lab makes some basic assumptions about your existing system:
+
+* Two Chef Servers
+  -  A Chef Server running some older version
+  -  A newly installed Chef Infra Server running the latest software, set up and configured for knife, but otherwise empty
+* One or more nodes
+  - Running Chef Infra Client 12-15
+  - Minimum 512 MB RAM
+  - [Recommended](https://docs.chef.io/chef_system_requirements/#chef-infra-client) 5GB space
+  - Bootstrapped to the older Chef Server
+  - `sudo` permissions on the node
+  - SSH connectivity between the nodes and the administrator/developer workstation
+* An administrator/developer workstation
+  - 64-bit architecture
+  - Minimum 4 GB RAM
+  - Recommended 10 GB of available disk space for installing Chef Workstation and running the Chef Upgrade Lab
+
+### Software Requirements
+
+* Meet the [platform and system requirements](https://docs.chef.io/workstation/install_workstation/) for Chef Workstation
+* Install or upgrade to the Chef Workstation [latest verision](https://downloads.chef.io/chef-workstation)
```

Comment:
```suggestion
* Install or upgrade to the Chef Workstation [latest version](https://downloads.chef.io/chef-workstation)
```

Reply:
Additionally, this reads somewhat awkwardly. I might change it to something like:

```suggestion
* Install or upgrade to the [latest version of Chef Workstation](https://downloads.chef.io/chef-workstation)
```